webui-src/app/people/people_util.js

```diff
@@ -1,13 +1,43 @@
-const m = require('mithril');
 const rs = require('rswebui');
-
+const m = require('mithril');
 function checksudo(id) {
   return id === '0000000000000000';
 }
 
+const UserAvatar = () => ({
+  view: (v) => {
+    const imageURI = v.attrs.avatar;
+    console.log(imageURI);
+    return imageURI === undefined || imageURI.mData.base64 === ''
+      ? []
+      : m('img.avatar', {
+          src: 'data:image/png;base64,' + imageURI.mData.base64,
+        });
+  },
+});
+
+function contactlist(list) {
```

Comment:
Better to have a more descriptive function name so it's clear what it does. Something like `regularContactList` or similar.

Reply:
Okk @rottencandy, I will update it.

Reply:
updated